### ISSUE TITLE
addition over erasure

### ISSUE THREAD

**Issue by hcayless** [sjhuskey/xml_patterns#1]
https://github.com/sjhuskey/xml_patterns/blob/master/XML_Patterns.xml#L62 should probably track what EpiDoc does. See http://www.stoa.org/epidoc/gl/latest/trans-struckovererasure.html, which would make it:

~~~~~
<rdg wit="#N" type="inRas"><add place="overstrike" hand="#N2">nocturna</add>que</rdg>
~~~~~

**Comment by sjhuskey** [comment]
That seems sensible. I wonder, though, if this is a place where the epigraphic origins of EpiDoc will be an obstacle to those working on manuscripts, etc. In other words, "overstrike" fits in epigraphy, but it doesn't describe what's going on in a manuscript.

**Comment by hcayless** [comment]
Yeah, maybe so. What would be a better alternative? We can change the suggested value list easily.

**Comment by sjhuskey** [comment]
Ah. I see the issue now. Thanks for the clarification. Would the existing value "inline" work, since we're indicating the erasure with @type on the reading?

I'll clean up the "types" document and give it home in the repository soon.

**Comment by hcayless** [comment]
I wondered about that...maybe. The problem with "inline" is that it might equally well mean ~~whatever~~nocturna (i.e. with the text added immediately after the erasure rather than on top of it).

**Comment by sjhuskey** [comment]
I was hoping that @type on the reading would clarify that issue. If it's a reading in rasura, that should mean that it is in the space where the original was erased.
